src/index.js

```diff
@@ -0,0 +1,58 @@
+import './style.css';
+
+const api = {
+  key: '80068fbfc8f92310ac9ea24006b4d8f7',
+  base: 'http://api.openweathermap.org/data/2.5/',
+};
+
+const inputValue = document.querySelector('.inputValue');
+inputValue.addEventListener('keypress', setQuery);
+
+function setQuery(e) {
+  if (e.keyCode === 13) {
+    getResults(inputValue.value);
+  }
+}
+
+function getResults(query) {
```

Comment:
Ought to use ES6 arrow functions instead of the `function` keyword to create functions